src/CMakeLists.txt

```diff
@@ -261,6 +261,7 @@ if(UNIX)
 		set_target_properties(mosquitto PROPERTIES
 			LINK_FLAGS "-Wl,-exported_symbols_list -Wl,${mosquitto_SOURCE_DIR}/src/linker-macosx.syms"
 		)
+	elseif (${CMAKE_SYSTEM_NAME} MATCHES "AIX")
```

Comment:
shouldn't you be adding your linker-aix.syms here too?  (Or make it emit a warning that cmake isn't supported on AIX?)

Reply:
`cmake` seems to generate an export file and adds some AIX export flags on its own. Using `make VERBOSE=1` I noticed:
```
-Wl,-bE:CMakeFiles/libmosquitto.dir/objects.exp
```
I have no idea how `cmake` handles this stuff but the build artifacts seem to work just the same.

Reply:
cool :) I'm not a cmake person either, just looked wrong here